packages/rfw/lib/src/dart/model.dart

```diff
@@ -439,6 +439,26 @@ class ConstructorCall extends BlobNode {
   String toString() => '$name($arguments)';
 }
 
+/// Representation of functions that return widgets in Remote Flutter library blobs.
+class WidgetBuilderDeclaration extends BlobNode {
+  /// Creates a [WidgetBuilderDeclaration].
```

Comment:
this documentation is insufficient per our style guide which says that documentation needs to have more information than can be guessed from just the name of the member being documented.
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc

Reply:
PTAL